lockFileLint.js

```diff
@@ -4,9 +4,15 @@ const fs = require('fs');
 
 const invalidations = [];
 const defaultLockfilePath = `${process.cwd()}/package-lock.json`;
+const exampleLockfilePath = `${process.cwd()}/examples/pirate-ship-app/package-lock.json`;
```

Comment:
What happens when we add more examples? Can you figure out a way that is similar to how we did the `packages/` directory?

Reply:
It may be slower but we could use `glob` to capture all lock files.

Something like: 

```
glob('**/package-lock.json', { ignore: '**/node_modules/**/package-lock.json' }, (e, files) => {
  if (e) {
    invalidations.push(e);
  }

  files.forEach(lockPath => {
    const { stderr } = spawnSync('./node_modules/.bin/lockfile-lint', [
      '-p',
      lockPath,
      '-t',
      'npm',
      '-a',
      'npm',
      '-o',
      'https:',
      '-c',
      '-i',
    ]);
    const error = stderr.toString();
    if (error) invalidations.push([lockPath, error].join(':\n\n'));
  });
});
```

Reply:
Yeah this is actually the functionality that we are wanting to add to `lockfile-lint` itself https://github.com/lirantal/lockfile-lint/issues/47

Reply:
I've updated with the latest commit. Will squash after review.